tests/unit/ContainerAutowireTest.php

```diff
@@ -2,6 +2,15 @@
 
 declare(strict_types=1);
```

Comment:
@Asisyas, this file is located in `unit` folder, whereas all other unit test classes are in `Unit` folder. Is it a mistake or intentially?

Reply:
That's my mistake. I'll fix it in the next commit.